package.yaml

```diff
@@ -30,7 +30,6 @@ dependencies:
   - temporary >= 1.3 && < 2
   - process >= 1.6.8 && < 2
   - free >= 5.1 && < 6
-  - transformers >= 0.5.6.2 && < 0.6
```

Comment:
I agree with having as few dependencies as possible. I didn't know about the `lift` vs `liftIO` trick :)

Reply:
You're welcome 😃.